node.go

```diff
@@ -246,6 +250,30 @@ func (n *node) collectedCount() (count int64) {
 	return
 }
 
+func (n *node) emittedCount() (count int64) {
+	for _, out := range n.outs {
+		count += out.collectedCount()
```

Comment:
Yep
On Mar 18, 2016 11:55 AM, "Yosi Attias" <notifications@github.com> wrote:

> In node.go
> <https://github.com/influxdata/kapacitor/pull/323#discussion_r56696878>:
>
> > @@ -246,6 +250,30 @@ func (n *node) collectedCount() (count int64) {
> >  	return
> >  }
> >
> > +func (n *node) emittedCount() (count int64) {
> > +	for _, out := range n.outs {
> > +		count += out.collectedCount()
>
> Just to make sure:
> node collected count is the sum of emitted counts of parent edges above
> collectedCount
> node emitted count is the sum of collected counts of children edges above
> emittedCount
>
> ?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/influxdata/kapacitor/pull/323/files/19d0ae1ab9e484ab4420b8f8901013506cb50800#r56696878>
>